server/schema/routes.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE routes (
+  id int PRIMARY KEY,
```

Comment:
to avoid having to calculate the next row ID when we insert new rows let's make `id` a `SERIAL` instead of an `INT`